tldw_Server_API/app/core/MCP_unified/server.py

```
@@ -895,7 +895,7 @@ async def handle_websocket(
                         metadata["roles"] = token_data.roles
                     if token_data.permissions:
                         metadata["permissions"] = token_data.permissions
-                except _MCP_SERVER_NONCRITICAL_EXCEPTIONS as _e:
+                except Exception as _e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)